Draft.md

```diff
@@ -192,128 +197,130 @@ Die folgenden Fragen nehmen Bezug auf technische Merkmale, welche die Nutzererfa
+Die Nutzbarkeit von Software wird von den Möglichkeiten der Kommunikation von Mensch und Programm bestimmt, also der Benutzeroberfläche (User Interface, UI). Diese ist in manchen Fällen grafisch gestützt (GUI), in anderen Fällen erfolgt die Bedienung über eine Befehlseingabe in der Konsole. Eine gemischte Bedienung ist ebenfalls nützlich, um verschiedene Nutzerkreise zu erreichen. Die Gestaltung von Oberflächen und die Nutzerführung in Menüstrukturen sind ein eigenes Aufgaben- und Forschungsfeld in der Softwareentwicklung. Gut an den Nutzer angepasste Lösungen sind das Ergebnis einer genauen Kenntnis der Zielgruppe und ihrer Gewohnheiten und Bedürfnisse, fügen sich also beispielsweise nahtlos in den archäologischen Forschungsprozess ein. Ein gutes Interface unterstützt das fehlerfreie und effiziente Arbeiten. So sind z. B. die Verständlichkeit der Menü-Einträge oder der Kommandos, aber auch die Aussagekraft von Fehlermeldungen Aspekte, die in der Rezension betrachtet werden sollten. Barrierefreiheit ist bislang ein unzureichend abgedecktes Kriterium, das aber ebenfalls die Effizienz und den möglichen Nutzerkreis adressiert.
 
-* **Passt das User Interface zum Nutzerkreis?** Jeder Nutzerkreis, auch der des Rezensenten, wird bestimmte Erwartungen an die Bedienbarkeit der Anwendung haben. Für die einen ist eine Kommandozeilenanwendung sehr gut zu bedienen, während andere damit Probleme haben werden. Viele Softwarelösungen haben mehrere Nutzerschnittstellen. Beispielsweise können viele Webanwendungen sowohl über einen Suchschlitz und Filterfunktionen über die grafische Oberfläche einer Website, als auch code-basiert über eine REST-Schnittstelle angesteuert werden.
+* **Passt das User Interface zum Nutzerkreis?** Jeder Nutzerkreis, auch der des Rezensenten, wird bestimmte Erwartungen an die Bedienbarkeit der Anwendung haben. Für die einen ist eine Kommandozeilenanwendung sehr gut zu bedienen, während andere damit Probleme haben werden. Viele Softwarelösungen haben mehrere Nutzerschnittstellen. Beispielsweise können viele Webanwendungen sowohl über einen Suchschlitz und Filterfunktionen in der grafischen Oberfläche einer Webseite als auch code-basiert über eine REST-Schnittstelle angesteuert werden.
```

Comment:
> Mir ist nicht ganz klar, wie sich der angesprochene Nutzerkreis definiert, bzw. wodurch er definiert wird. Zudem ist mir die Bezeichnung ‚Suchschlitz‘ nicht geläufig.

ad Suchschlitz: eigentlich doch selbsterklärend? Sonst lieber 'Suchfeld'?

Reply:
Guter Vorschlag. Wikipedia weiß gar, "Im Kontext von elektronischen Suchanfragen, also an elektronische Datenverarbeitungsanlagen, Datenbanken oder Suchmaschinen gerichtete Aufträge zur Durchsuchung des jeweiligen Datenbestands nach spezifischen Suchbegriffen oder Suchmasken, spricht man auch von einem Suchfeld. Dieses kann Teil einer Suchleiste sein." Seite „Textfeld“. In: Wikipedia, Die freie Enzyklopädie. Bearbeitungsstand: 21. August 2020, 21:11 UTC. URL: https://de.wikipedia.org/w/index.php?title=Textfeld&oldid=202981993 (Abgerufen: 21. November 2020, 18:50 UTC)

Reply:
Suchfeld eingefügt; 
Die Frage nach dem Nutzerkreis ist noch offen

Reply:
Ich glaube, wir müssen einfach das Wort Nutzerkreis etwas greifbarer machen. Vielleicht können wir den einleitenden Abschnitt von **Interface** etwas erweitern. Nach *um verschiedene Nutzerkreise zu erreichen.* und vor *Die Gestaltung von Oberflächen und die Nutzerführung in Menüstrukturen* würde ich folgendes einfügen:

> Die sind natürlich vielfältig und niemals scharf umgrenzt, doch sicher haben Feldarchäologen und Grabungstechniker andere Anforderungen als Kollegen in der Denkmalverwaltung oder explizite Archäoinformatiker.

Reply:
Vielleicht einfach "Jede Nutzergruppe"?
Wir wollten ja eigentlich kürzen, und ich versteh das Problem der*s Resenzent*in hier eher als "ich kenn das Wort "Nutzerkreis"" nicht. Aber vielleicht irre ich mich auch und dann fänd ich den Vorschlag von @nevrome gut. Würde nur das Wort "sicher" weglassen.

Reply:
Ich finde auch den Vorschlag von @nevrome gut. Meine Intention als ich das geschrieben habe war eher, dass der Rezensent aufgrund seines Hintergrunds für eine Gruppe von Nutzern  stellvertretend die Anwendung beurteilen kann, ggf. aber auch die Bedienungspräferenzen für andere Nutzer antizipieren kann.

Reply:
Habe mal statt Nutzerkreis Nutzergruppe genommen und auch @nevrome s Vorschlag aufgegriffen und in der Interface-Einleitung folgenden Satz eingebaut:

> Eine gemischte Bedienung ist ebenfalls nützlich, um verschiedene Nutzergruppen anszusprechen, welche oft vielfältig und nie scharf umgrenzt werden können, wie etwa Grabungstechniker oder Landesarchäologen.